File: tools/reference/median-of-two-sorted-arrays.py
def findMedianSortedArrays(nums1, nums2):
    merged = sorted(nums1 + nums2)
    n = len(merged)
    mid = n // 2
    if n % 2:
        return float(merged[mid])
    return (merged[mid - 1] + merged[mid]) / 2.0

File: tools/reference/median-of-two-sorted-arrays.js
var findMedianSortedArrays = function (nums1, nums2) {
  const merged = [...nums1, ...nums2].sort((a, b) => a - b);
  const mid = Math.floor(merged.length / 2);
  return merged.length % 2 ? merged[mid] : (merged[mid - 1] + merged[mid]) / 2;
};
